src/Sylius/Bundle/CoreBundle/PaymentRequest/Payum/CommandHandler/ModelPaymentRequestHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\PaymentRequest\Payum\CommandHandler;
```

Comment:
I didn't read on detail yet what happens in this code, but `PaymentRequest` should have single command handler with "components" inside.

```suggestion
namespace Sylius\Bundle\CoreBundle\PaymentRequest\CommandHandler\Payum;
```